models/task.py

```diff
@@ -1,13 +1,13 @@
 from models.configuration import Configuration
 from models.notification import Notification
-from models.schedule import Schedule
+from schedule.factory import KindFactory
 
 
 class Task:
     def __init__(self, json_task):
         self.name = json_task["name"]
         self.kind = json_task["kind"]
-        self.schedule = Schedule(json_task["schedule"])
+        self.schedule = KindFactory(json_task["schedule"]).new()
```

Comment:
```suggestion
        self.schedule = ScheduleFactory(json_task["schedule"]).new()
```

Acho que fica mais claro do que se trata.. ou poderia manter só Factory e no import usar um alias